README.md

```
@@ -126,10 +126,106 @@ Cluster leaves which split close to zero, but not at it, predict one another ver
+# For newer matplotlib, ax might be the direct axis:
+# ax.tick_params(axis='y', labelsize=30)
+# plt.show() # if not in a Jupyter environment
+```
+
+## Contributing
+
+This library is currently being actively modernized and developed. Contributions for bug fixes, new features, and improvements are welcome!
+Please see the [Issues](https://github.com/ResidentMario/missingno/issues) page for project ideas, to report a problem, or to discuss a new feature.
+For more details on how to contribute, please see the `CONTRIBUTING.md` file.
+
+[matrix_filtered]: http://i.imgur.com/UF6hmL8.png
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Duplicate link or image reference definition: "matrix_filtered"

(MD053, link-image-reference-definitions)